code/datums/outfit/medical.dm

```diff
@@ -154,6 +161,12 @@
 
 /datum/outfit/doctor/pre_equip_priority(var/mob/living/carbon/human/H, var/species)
 	items_to_collect[/obj/item/weapon/storage/belt/medical] = GRASP_LEFT_HAND
+	
+	return ..()
+	
+/datum/outfit/doctor/post_equip_priority(var/mob/living/carbon/human/H)
+	var/obj/item/weapon/reagent_containers/food/drinks/soda_cans/randomcan = pick(/obj/item/weapon/reagent_containers/food/drinks/soda_cans/lifeline_white, /obj/item/weapon/reagent_containers/food/drinks/soda_cans/lifeline_red, /obj/item/weapon/reagent_containers/food/drinks/soda_cans/lifeline_cryo)
+	H.equip_or_collect(new randomcan, slot_in_backpack)
```

Comment:
```suggestion
	items_to_collect[randomcam] = GRASP_RIGHT_HAND // Because left one is already taken
```
And likewise for the rest
You can do the `pick` logic in `pre_equip_priority`. Post-equip should only be done for things that act on the human mob.